### ISSUE TITLE
[JENKINS-24142] Node selection option for a Label configured Job

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/nodelabelparameter-plugin#524]
<p>want to have a new feature for Node selection when we run a job with parameters, For example, we have a Job which is configured NodeLable parameter called "devbuilds" which is having 10 nodes associated to it,that means the job can run concurrently on 10 nodes based on the availability of the nodes from Lable but sometime we want to run this job on a specific node out of 10 nodes for debugging the build issues, so we need to edit the Job<br/>
config to run on a single node and run the job, this will be more time consuming process and all the dev team doesn't have the access to change the config,it would be nice if we have an option to switch between Lable and or Node before starting the Job.</p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=prab">prab</a>, imported from: <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-24142" target="_blank">Node selection option for a Label configured Job</a></i></summary>
<i><ul>
<li><b>status</b>: Closed
<li><b>priority</b>: Minor
<li><b>component(s)</b>: nodelabelparameter-plugin
<li><b>resolution</b>: Won't Fix
<li><b>resolved</b>: 2025-05-19T13:26:11+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 2
<li><b>imported</b>: 20251210-235911
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>want to have a new feature for Node selection when we run a job with parameters, For example, we have a Job which is configured NodeLable parameter called "devbuilds" which is having 10 nodes associated to it,that means the job can run concurrently on 10 nodes based on the availability of the nodes from Lable but sometime we want to run this job on a specific node out of 10 nodes for debugging the build issues, so we need to edit the Job
config to run on a single node and run the job, this will be more time consuming process and all the dev team doesn't have the access to change the config,it would be nice if we have an option to switch between Lable and or Node before starting the Job.</p></pre>
</details>
</details>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/danielbeck.png" title="danielbeck's avatar" /> danielbeck:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-24142?focusedId=207329&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-207329">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Issue description is very difficult to understand. Please try to use standard English grammar and punctuation, and fix that run-on sentence in the first paragraph.</p>

<p>Issue appears to not a bug. Please explain in detail how this issue constitutes a bug in the NodeLabel parameter plugin.</p>

<p>Note that if you're asking for assistance rather than reporting a bug, Jira is the wrong place to do so. Try the jenkinsci-users mailing list or #jenkins IRC channel on Freenode instead.</p>
</pre>
</details>

</ul>
</details>

<p>Issue description is very difficult to understand. Please try to use standard English grammar and punctuation, and fix that run-on sentence in the first paragraph.</p>

<p>Issue appears to not a bug. Please explain in detail how this issue constitutes a bug in the NodeLabel parameter plugin.</p>

<p>Note that if you're asking for assistance rather than reporting a bug, Jira is the wrong place to do so. Try the jenkinsci-users mailing list or #jenkins IRC channel on Freenode instead.</p>
